main.py

```diff
@@ -1,11 +1,51 @@
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
+import json
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players:
+        data = json.load(players)
+        list_race = []
```

Comment:
do not overload context managers!
they are created for reading files
So, do your business logic outside of context managers
https://github.com/mate-academy/py-game-models/blob/master/checklist.md#1-do-not-overload-the-context-manager